app/src/main/java/com/puzzlebench/clean_marvel_kotlin/domain/model/Character.kt

```diff
@@ -1,7 +1,39 @@
 package com.puzzlebench.clean_marvel_kotlin.domain.model
 
+import android.os.Parcel
+import android.os.Parcelable
+
+
 class Character(
+        val id: Int,
         val name: String,
         val description: String,
         val thumbnail: Thumbnail
-)
+) : Parcelable {
```

Comment:
Hernan, sorry, but why you use a Parcelable obj here?, I think that its not necessary.

Reply:
Modified. In next commit Character class dont extend Parcelable.